configuration/src/main/java/pl/edu/uj/kimage/configuration/ConfigurationReader.java

```diff
@@ -0,0 +1,90 @@
+package pl.edu.uj.kimage.configuration;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Lists;
+
+import javax.annotation.concurrent.ThreadSafe;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+@ThreadSafe
+public class ConfigurationReader {
+    private static final ConfigurationReader INSTANCE = new ConfigurationReader();
+    private static final int KEY_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+    private static final String CONFIG_FILE = "config.properties";
```

Comment:
cause now there is only one use case, if there will be need for more then it could be passed as method parameter